config/datocms/migrations/1741855116_addGroupingBlock.ts

```diff
@@ -0,0 +1,224 @@
+    appearance: {
+      addons: [],
+      editor: 'single_line',
+      parameters: { heading: false, placeholder: null },
+    },
+    default_value: null,
+  });
+
+  console.log(
+    'Create Modular Content (Multiple blocks) field "Body" (`blocks`) in block model "\uD83D\uDDC2\uFE0F Grouping Item Block" (`grouping_item_block`)',
+  );
+  await client.fields.create('BeM4dW2OQYWKc9iBZUyMeg', {
+    id: 'NTDc3vtCRzO5mEsE3gfmOQ',
+    label: 'Body',
+    field_type: 'rich_text',
+    api_key: 'blocks',
+    validators: {
+      rich_text_blocks: {
+        item_types: [
+          'BRbU6VwTRgmG5SbwUs0rBg',
+          'F60ZY1wFSW2qbvh99poj3g',
+          'PAk40zGjQJCcDXXPgygUrA',
+          'QYfZyBzIRWKxA1MinIR0aQ',
+          'ZdBokLsWRgKKjHrKeJzdpw',
+          'gezG9nO7SfaiWcWnp-HNqw',
+          '0SxYNS2CR1it_5LHYWuEQg',
+        ],
+      },
+      size: { min: 1 },
+    },
+    appearance: {
+      addons: [],
+      editor: 'rich_text',
+      parameters: { start_collapsed: false },
+    },
+    default_value: null,
+  });
+
+  console.log('Update existing fields/fieldsets');
+
+  console.log(
+    'Update Modular Content (Multiple blocks) field "Body" (`blocks`) in model "\uD83E\uDDE9 Page Partial" (`page_partial`)',
+  );
+  await client.fields.update('SKLmdv71Rge0rKhJzOFQWQ', {
+    validators: {
+      rich_text_blocks: {
+        item_types: [
+          'BRbU6VwTRgmG5SbwUs0rBg',
+          'F60ZY1wFSW2qbvh99poj3g',
+          'PAk40zGjQJCcDXXPgygUrA',
+          'TBuD6qQOSDy6i9dM3T_XEA',
+          'ZdBokLsWRgKKjHrKeJzdpw',
+          'gezG9nO7SfaiWcWnp-HNqw',
+          '0SxYNS2CR1it_5LHYWuEQg',
+        ],
+      },
+    },
+  });
+
+  console.log(
+    'Update Modular Content (Multiple blocks) field "Body" (`body_blocks`) in model "\uD83D\uDCD1 Page" (`page`)',
+  );
+  await client.fields.update('Q-z1nyMsQtC8Sr6w6J2oGw', {
+    validators: {
+      rich_text_blocks: {
+        item_types: [
+          'BRbU6VwTRgmG5SbwUs0rBg',
+          'F60ZY1wFSW2qbvh99poj3g',
+          'PAk40zGjQJCcDXXPgygUrA',
+          'QYfZyBzIRWKxA1MinIR0aQ',
+          'TBuD6qQOSDy6i9dM3T_XEA',
+          'VZvVfu52RZK81WG0Dxp-FQ',
+          'V80liDVtRC-UYgd3Sm-dXg',
+          'ZdBokLsWRgKKjHrKeJzdpw',
+          'gezG9nO7SfaiWcWnp-HNqw',
+          '0SxYNS2CR1it_5LHYWuEQg',
+        ],
+      },
+    },
+  });
+
+  console.log(
+    'Update Modular Content (Multiple blocks) field "Body" (`body_blocks`) in model "\uD83C\uDFE0 Home" (`home_page`)',
+  );
+  await client.fields.update('pUj2PObgTyC-8X4lvZLMBA', {
+    validators: {
+      rich_text_blocks: {
+        item_types: [
+          'BRbU6VwTRgmG5SbwUs0rBg',
+          'F60ZY1wFSW2qbvh99poj3g',
+          'PAk40zGjQJCcDXXPgygUrA',
+          'QYfZyBzIRWKxA1MinIR0aQ',
+          'TBuD6qQOSDy6i9dM3T_XEA',
+          'VZvVfu52RZK81WG0Dxp-FQ',
+          'V80liDVtRC-UYgd3Sm-dXg',
+          'ZdBokLsWRgKKjHrKeJzdpw',
+          'gezG9nO7SfaiWcWnp-HNqw',
+          '0SxYNS2CR1it_5LHYWuEQg',
```

Comment:
Consider writing this in a nondestructive way. See https://github.com/voorhoede/head-start/pull/245

For this migration I'd say do the following for pages and homepage:

```
const homePageBodyBlocks = (await client.fields.find("home_page::body_blocks"))
  .validators?.rich_text_blocks?.item_types || [];

await client.fields.update('pUj2PObgTyC-8X4lvZLMBA', {
  validators: {
    rich_text_blocks: {
      item_types: [
        ...homePageBodyBlocks,
        //  grouping block.
```